Source/WebKit/GPUProcess/graphics/RemoteRenderingBackend.cpp

```diff
@@ -369,9 +372,7 @@ void RemoteRenderingBackend::cacheFontCustomPlatformData(WebCore::FontCustomPlat
 {
     ASSERT(!RunLoop::isMain());
 
-    // FIXME: (rdar://124235570) use this->shoulUsedLockdownFontParser instead of hard-coded 'false' at tryMakeFromSerializationData after we deprecate lockdown mode fonts allowed (trusted) list.
-    constexpr bool shouldUseLockdownFontParser { false };
-    auto customPlatformData = FontCustomPlatformData::tryMakeFromSerializationData(WTFMove(fontCustomPlatformSerializedData), shouldUseLockdownFontParser);
+    auto customPlatformData = FontCustomPlatformData::tryMakeFromSerializationData(WTFMove(fontCustomPlatformSerializedData), this->shouldUseLockdownFontParser());
```

Comment:
Nit: I don't think `this` is necessary anymore, now that the overloaded `shouldUseLockdownFontParser` local has been removed.